README.md

```diff
@@ -3,3 +3,171 @@
 Jupyter extensions and magics for working with remote Dataproc clusters with
 Livy and Component Gateway.
 
+## Before you begin
+
+In order to use this library, you first need to go through the following steps:
+
+1. [Select or create a Cloud Platform project][create_project]
+2. [Enable billing for your project][enable_billing]
+3. [Enable the Google Cloud Dataproc API][enable_api]
+4. [Setup Authentication][authentication]
+
+[create_project]: https://console.cloud.google.com/project
+[enable_billing]: https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project
+[enable_api]: https://cloud.google.com/dataproc
+[authentication]: https://cloud.google.com/docs/authentication/getting-started#auth-cloud-implicit-python
+
+
+## Installation
+
+To install into a Jupyter notebook running locally:
+
+1. Install the google-cloud-dataproc Cloud Client Libraries.
+
+    ```bash
+    pip install google-cloud-dataproc==2.0.0 --force-reinstall --no-dependencies
```

Comment:
Out of curiosity, does this not get pulled in automatically by pip installing this repo?